platforms/chibios/chibios_config.h

```diff
@@ -42,8 +42,12 @@
 
 #    define usb_lld_endpoint_fields
 
-#    define I2C1_SCL_PAL_MODE (PAL_MODE_ALTERNATE_I2C | PAL_RP_PAD_SLEWFAST | PAL_RP_PAD_PUE | PAL_RP_PAD_DRIVE4)
-#    define I2C1_SDA_PAL_MODE I2C1_SCL_PAL_MODE
+#    if !defined(I2C1_SCL_PAL_MODE)
```

Comment:
This should be removed from the PR and moved to a separate PR

Reply:
it actually already is: #19286 
... which is stuck and waiting for reviews and/or clear vetos :-)